### PR TITLE
promote resource privileged_access_manager_entitlement to ga

### PR DESCRIPTION
**Pull request by varsharmavs**
```release-note:enhancement
privilegedaccessmanager: promoted `google_privileged_access_manager_entitlement` resource from beta to ga
```

### CODE REVIEW

**Review by roaks3 — CHANGES_REQUESTED** [review]
I think you need to change the `ProtoV5ProviderBetaFactories` line in your update test. The error indicates that multiple versions of the provider are trying to be used.

File: mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb [hunk 1]
```diff
@@ -20,4 +20,4 @@ func validateEntitlementId(v interface{}, k string) (ws []string, errors []error
     }
 
     return
-}
```
Comment (gouthamkrishna) [inline mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb:23]: How is this working now? If it is not working, why is there no tests that are failing?

File: mmv1/templates/terraform/examples/privileged_access_manager_entitlement_basic.tf.erb [hunk 2]
```diff
@@ -21,17 +22,27 @@ resource "google_privileged_access_manager_entitlement" "<%= ctx[:primary_resour
         }
     }
     additional_notification_targets {
-    admin_email_recipients     = ["user@example.com"]
-    requester_email_recipients = ["user@example.com"]
+      admin_email_recipients     = [
+        "user@example.com",
+        "user@example.com"
```
Comment (gouthamkrishna) [inline mmv1/templates/terraform/examples/privileged_access_manager_entitlement_basic.tf.erb:27]: remove the repetition?

File: mmv1/products/privilegedaccessmanager/product.yaml [hunk 3]
```diff
@@ -14,8 +14,11 @@
 --- !ruby/object:Api::Product
 versions:
   - !ruby/object:Api::Product::Version
-    base_url: https://privilegedaccessmanager.googleapis.com/v1beta/
+    name: ga
+    base_url: https://privilegedaccessmanager.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
     name: beta
+    base_url: https://privilegedaccessmanager.googleapis.com/v1/
```
Comment (gouthamkrishna) [inline mmv1/products/privilegedaccessmanager/product.yaml:21]: This should be v1beta

File: mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb (hunk 1 above)
Comment (gouthamkrishna) [reply mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb:23]: I see that validateEntitlementId func was not closed. Is it working now? (without adding this closing curly braces)

Comment (varsharmavs) [reply mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb:23]: The function is closed Goutham, There must be some space or newline change for which git is showing us a diff.

**Review by gouthamkrishna — APPROVED** [review]

**Review by roaks3 — APPROVED** [review]

**Pull request merged**